pymaker/oasis.py

```diff
@@ -629,8 +629,9 @@ def get_orders(self, pay_token: Address = None, buy_token: Address = None) -> Li
         Returns:
             A list of `Order` objects representing all active orders on Oasis.
         """
-        assert((isinstance(pay_token, Address) and isinstance(buy_token, Address))
-               or (pay_token is None and buy_token is None))
+    
+        assert((isinstance(pay_token, Address) and isinstance(p_token, Token) and isinstance(buy_token, Address) and isinstance(b_token, Token))
```

Comment:
this won't be needed anymore if passing only Token and retrieving Address from it